src/common/session/session-entity.js

```diff
@@ -39,13 +39,18 @@ export class SessionEntity {
     if (!isBrowserScope) this.storage = new LocalMemory()
     else this.storage = storageAPI
 
+    this.sync(model)
```

Comment:
Should `inactiveAt` and `expiresAt` also default to meaningful values here, if those properties are getting unnecessarily set once here and again in the following blocks?

Reply:
Probably.  This is mostly to make sure keys checks always have a property exist, but open to opinions on what those meaningful defaults would be for those timestamps